project/templates/Friendspage.html

```diff
@@ -0,0 +1,258 @@
+
+<html>
+    <head> 
+
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1 ">
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-iYQeCzEYFbKjA/T2uDLTpkwGzCiq6soy8tYaI1GyVh/UjpbCx/TYkiZhlZB6+fzT" crossorigin="anonymous">

```

Comment:
Try to take out the head stuff (bootstrap, fontawesome, etc) to the base.html file and use the {% extend %} thing to get everything from base.html in the friends page. Check to see if it breaks the other pages though.